### ISSUE TITLE
Chart 15.2.x Forces namespacing

### ISSUE THREAD

**Issue by nniehoff** [traefik/traefik-helm-chart#672]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik-helm-chart/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What version of the Traefik's Helm Chart are you using?

15.2.0

### What version of Traefik are you using?

2.9.1

### What did you do?

I upgraded chart versions from 15.1.1 to 15.2.0. (actually, I tried 15.2.2 first but have narrowed down the breaking change to 15.2.0)

### What did you see instead?

With 15.1.1 the parameter `--providers.kubernetesingress.namespaces` does not exist in the pod spec.  In 15.2.0 with the same values the parameter `--providers.kubernetesingress.namespaces=traefik` is added.

### What is your environment & configuration?

```yaml
additionalArguments:
- --entryPoints.traefik.transport.respondingTimeouts.idleTimeout=60
- --entryPoints.web.transport.respondingTimeouts.idleTimeout=60
- --entryPoints.websecure.transport.respondingTimeouts.idleTimeout=60
- --providers.kubernetesingress.allowexternalnameservices=true
- --serversTransport.insecureSkipVerify=true
affinity:
  podAntiAffinity:
    requiredDuringSchedulingIgnoredDuringExecution:
    - labelSelector:
        matchLabels:
          app.kubernetes.io/instance: traefik
          app.kubernetes.io/name: traefik
      topologyKey: kubernetes.io/hostname
    - labelSelector:
        matchLabels:
          app.kubernetes.io/instance: traefik
          app.kubernetes.io/name: traefik
      topologyKey: topology.kubernetes.io/zone
deployment:
  minReadySeconds: 210
  replicas: 3
ingressClass:
  enabled: true
  isDefaultClass: true
logs:
  access:
    enabled: true
    format: json
  general:
    format: json
    level: DEBUG
podDisruptionBudget:
  enabled: true
  minAvailable: 1
ports:
  websecure:
    tls:
      enabled: true
providers:
  kubernetesCRD:
    allowExternalNameServices: true
    labelSelector: '!endpoint'
  kubernetesIngress:
    allowExternalNameServices: true
    labelSelector: '!endpoint'
resources:
  requests:
    cpu: 30m
    memory: 128Mi
rollingUpdate:
  maxSurge: 0
  maxUnavailable: 1
service:
  enabled: false
```

**Comment by nniehoff** [comment]
As a workaround I with chart 15.2.2 I have set `.Values.providers.kubernetesIngress.namespaces=null`

**Comment by connec** [comment]
This should be fixed in 15.2.2. You might need to `helm repo update` to definitely pull the latest chart (I just pranked myself with this).
